charts/tidb-cluster/values.yaml

```diff
@@ -15,6 +15,9 @@ timezone: Asia/Shanghai
 # default reclaim policy of a PV
 pvReclaimPolicy: Retain
 
+# enable tidb binlog
+binlogEnabled: false
```

Comment:
Pump and Drainer are related to binlog, and actually, they are components of binlog. So I think they should be organized as Prometheus and Grafana to `monitor`.